# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-contain/reference/quote-scoping-002-ref.html
# Commit: e25543fae7ee
# Full Hash: e25543fae7ee03ac57b9cfbfeb2418c9e2f490fc
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r 4230bdcf07ae -r e25543fae7ee testing/web-platform/tests/css/css-contain/reference/quote-scoping-002-ref.html
--- a/testing/web-platform/tests/css/css-contain/reference/quote-scoping-002-ref.html	Wed Jun 22 16:15:57 2022 +0000
+++ b/testing/web-platform/tests/css/css-contain/reference/quote-scoping-002-ref.html	Wed Jun 22 16:16:59 2022 +0000
@@ -4,5 +4,5 @@
   <title>CSS-contain test reference</title>
   <link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net">
 
-<p>Test passes if the text below is "A9Z" (not including the quotation marks).<p>
-<div>A9Z</span></div>
+<p>Test passes if the text below is "AZZ" (not including the quotation marks).<p>
+<div>AZZ</span></div>
